src/transform/benches/README.md

```diff
@@ -0,0 +1,83 @@
+# `mzbench-opt`
+
+The `mzbench-opt` package contains optimizer benchmarking scripts.
+
+## Installation (`pyenv`)
+
+Run the following commands only once on each host.
+
+```bash
+pyenv virtualenv 3.9.5 mzbench-opt
```

Comment:
On macOS 12.0.1 with an M1 chip, I had trouble building v3.9.5 but v3.9.9 worked fine. We may want to bump to that version.

Reply:
I abstracted over the Python version with a `PYTHON_VERSION` variable in the scripts and noted that we need something higher than `3.9.0` (although lower might also be fine, I just have not checked that).